iConverter/Pages/Base/BaseInteractor.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  BaseInteractor.swift
+//  iConverter
+//
+//  Created by Aram Semerjyan on 13.04.22.
+//
+
+import RxSwift
+
+class BaseInteractor {
+    
+    deinit {
+        print("********")
+        print("Interactor deinited")
+        print("********")
+    }
+
+    var disposeBag = DisposeBag()
```

Comment:
Why `var` ?

Reply:
That was added from my first approaches. I'll remove it